internal/pkg/externalplugins/label/label.go

```diff
@@ -107,29 +113,36 @@
 
 // getLabelsFromGenericMatches returns label matches with extra labels if those
 // have been configured in the plugin config.
-func getLabelsFromGenericMatches(matches [][]string, additionalLabels []string) []string {
+func getLabelsFromGenericMatches(matches [][]string, additionalLabels []string, invalidLabels *[]string) []string {
 	if len(additionalLabels) == 0 {
 		return nil
 	}
+
 	var labels []string
+	labelFilter := sets.String{}
```

Comment:
I don't think this is a good name, because in general, a filter should be a method rather than a collection.

Reply:
Indeed, do you have another idea about the name?

Here is a reference from https://github.com/kubernetes/test-infra/blob/e9bafa1cc6be4c8fae4046cef5aeccaa28b3a648/prow/plugins/label/label.go#L117

Reply:
How about additionalLabelsSet?

Reply:
Maybe labelFilter is fine.